ca.lproj/Localizable.strings

```diff
@@ -153,8 +153,62 @@
 "WATCH_LIBRARY_SYNC_IN_PROGRESS" = "Sincronitzant…";
 
 "ALERT_IMPORT" = "Importa";
-"IMPORT_TITLE" = "¿Importa \"%@\"?";
-"IMPORT_BODY" = "Aquesta col·lecció compartida conté% ld emissores.";
+"IMPORT_TITLE" = "Importa \"%@\"?";
+"IMPORT_BODY" = "Aquesta col·lecció compartida conté %ld emissores.";
 
-"MOVE_DIALOG_TITLE" = "Moure \"%@\"";
+"MOVE_DIALOG_TITLE" = "Mou \"%@\"";
 "MOVE_DIALOG_BODY" = "Trieu una col·lecció a on moure aquesta emissora.";
+
+/* Broadcasts v2.0 */
+
+"ERROR_ALERT_TITLE" = "Error: %@";
+
+"LIBRARY_STATE_UNAUTHENTICATED" = "Sessió no iniciada";
+"LIBRARY_STATE_QUOTA_EXCEEDED" = "Sense espai";
+
+"SIDEBAR_CATEGORY_DIRECTORY" = "Explora";
+"SIDEBAR_TITLE_DIRECTORY" = "Directori";
+
+"EDIT_STATION_WINDOW_INTERNET_RADIO_PLACEHOLDER" = "Ràdio en línia";
+
+"STATION_METADATA_NAME_PLACEHOLDER" = "Emissora sense nom";
+
+"WELCOME_BUTTON_CONTINUE" = "Continua";
+"WELCOME_BUTTON_GETSTARTED" = "Comença";
+
+"WELCOME_NEWFEATURES_TITLE" = "Noves funcions";
+
+"WELCOME_NEWFEATURES_2_DESIGN" = "Aparença moderna";
+"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "Temporitzador";
+"WELCOME_NEWFEATURES_2_AIRPLAY" = "Controls en la barra d'AirPlay";
+"WELCOME_NEWFEATURES_2_ALERTS" = "Alertes menys intrusives";
+"WELCOME_NEWFEATURES_2_NATIVESCALING" = "Escalat natiu";
```

Comment:
Just to check — this is how you would phrase it for the process of scaling an image, or rendering at high resolution?

Reply:
I think so, maybe "Resolució nativa" (_native resolution_) is better?